File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/ui/meteoriteRecyclerView/selector/MeteoriteSelector.java
package com.antonio.samir.meteoritelandingsspots.ui.meteoriteRecyclerView.selector;


import com.antonio.samir.meteoritelandingsspots.model.Meteorite;

/**
 * Meteorite Selector interface to abstract the selection command from Meteorite Adapter
 */
public interface MeteoriteSelector {

    void selectItemId(final Meteorite meteorite);

}

File: app/src/main/java/com/antonio/samir/meteoritelandingsspots/features/list/recyclerView/selector/MeteoriteSelector.kt
package com.antonio.samir.meteoritelandingsspots.features.list.recyclerView.selector

import com.antonio.samir.meteoritelandingsspots.data.repository.model.Meteorite


/**
 * Meteorite Selector interface to abstract the selection command from Meteorite Adapter
 */
interface MeteoriteSelector {

    fun selectItem(meteorite: Meteorite)

}
